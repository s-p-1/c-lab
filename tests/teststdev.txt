disable_output
A1=STDEV(A2:ZZ101)
A2=STDEV(A3:ZZ102)
A3=STDEV(A4:ZZ103)
A4=STDEV(A5:ZZ104)
A5=STDEV(A6:ZZ105)
A6=STDEV(A7:ZZ106)
A7=STDEV(A8:ZZ107)
A8=STDEV(A9:ZZ108)
A9=STDEV(A10:ZZ109)
A10=STDEV(A11:ZZ110)
A11=STDEV(A12:ZZ111)
A12=STDEV(A13:ZZ112)
A13=STDEV(A14:ZZ113)
A14=STDEV(A15:ZZ114)
A15=STDEV(A16:ZZ115)
A16=STDEV(A17:ZZ116)
A17=STDEV(A18:ZZ117)
A18=STDEV(A19:ZZ118)
A19=STDEV(A20:ZZ119)
A20=STDEV(A21:ZZ120)
A21=STDEV(A22:ZZ121)
A22=STDEV(A23:ZZ122)
A23=STDEV(A24:ZZ123)
A24=STDEV(A25:ZZ124)
A25=STDEV(A26:ZZ125)
A26=STDEV(A27:ZZ126)
A27=STDEV(A28:ZZ127)
A28=STDEV(A29:ZZ128)
A29=STDEV(A30:ZZ129)
A30=STDEV(A31:ZZ130)
A31=STDEV(A32:ZZ131)
A32=STDEV(A33:ZZ132)
A33=STDEV(A34:ZZ133)
A34=STDEV(A35:ZZ134)
A35=STDEV(A36:ZZ135)
A36=STDEV(A37:ZZ136)
A37=STDEV(A38:ZZ137)
A38=STDEV(A39:ZZ138)
A39=STDEV(A40:ZZ139)
A40=STDEV(A41:ZZ140)
A41=STDEV(A42:ZZ141)
A42=STDEV(A43:ZZ142)
A43=STDEV(A44:ZZ143)
A44=STDEV(A45:ZZ144)
A45=STDEV(A46:ZZ145)
A46=STDEV(A47:ZZ146)
A47=STDEV(A48:ZZ147)
A48=STDEV(A49:ZZ148)
A49=STDEV(A50:ZZ149)
A50=STDEV(A51:ZZ150)
A51=STDEV(A52:ZZ151)
A52=STDEV(A53:ZZ152)
A53=STDEV(A54:ZZ153)
A54=STDEV(A55:ZZ154)
A55=STDEV(A56:ZZ155)
A56=STDEV(A57:ZZ156)
A57=STDEV(A58:ZZ157)
A58=STDEV(A59:ZZ158)
A59=STDEV(A60:ZZ159)
A60=STDEV(A61:ZZ160)
A61=STDEV(A62:ZZ161)
A62=STDEV(A63:ZZ162)
A63=STDEV(A64:ZZ163)
A64=STDEV(A65:ZZ164)
A65=STDEV(A66:ZZ165)
A66=STDEV(A67:ZZ166)
A67=STDEV(A68:ZZ167)
A68=STDEV(A69:ZZ168)
A69=STDEV(A70:ZZ169)
A70=STDEV(A71:ZZ170)
A71=STDEV(A72:ZZ171)
A72=STDEV(A73:ZZ172)
A73=STDEV(A74:ZZ173)
A74=STDEV(A75:ZZ174)
A75=STDEV(A76:ZZ175)
A76=STDEV(A77:ZZ176)
A77=STDEV(A78:ZZ177)
A78=STDEV(A79:ZZ178)
A79=STDEV(A80:ZZ179)
A80=STDEV(A81:ZZ180)
A81=STDEV(A82:ZZ181)
A82=STDEV(A83:ZZ182)
A83=STDEV(A84:ZZ183)
A84=STDEV(A85:ZZ184)
A85=STDEV(A86:ZZ185)
A86=STDEV(A87:ZZ186)
A87=STDEV(A88:ZZ187)
A88=STDEV(A89:ZZ188)
A89=STDEV(A90:ZZ189)
A90=STDEV(A91:ZZ190)
A91=STDEV(A92:ZZ191)
A92=STDEV(A93:ZZ192)
A93=STDEV(A94:ZZ193)
A94=STDEV(A95:ZZ194)
A95=STDEV(A96:ZZ195)
A96=STDEV(A97:ZZ196)
A97=STDEV(A98:ZZ197)
A98=STDEV(A99:ZZ198)
A99=STDEV(A100:ZZ199)
A999=B1
enable_output
A99=10000
scroll_to A90
q

