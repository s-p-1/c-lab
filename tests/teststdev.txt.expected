              A           B           C           D           E           F           G           H           I           J
  1           0           0           0           0           0           0           0           0           0           0
  2           0           0           0           0           0           0           0           0           0           0
  3           0           0           0           0           0           0           0           0           0           0
  4           0           0           0           0           0           0           0           0           0           0
  5           0           0           0           0           0           0           0           0           0           0
  6           0           0           0           0           0           0           0           0           0           0
  7           0           0           0           0           0           0           0           0           0           0
  8           0           0           0           0           0           0           0           0           0           0
  9           0           0           0           0           0           0           0           0           0           0
 10           0           0           0           0           0           0           0           0           0           0
[0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) > [0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
  1           0           0           0           0           0           0           0           0           0           0
  2           0           0           0           0           0           0           0           0           0           0
  3           0           0           0           0           0           0           0           0           0           0
  4           0           0           0           0           0           0           0           0           0           0
  5           0           0           0           0           0           0           0           0           0           0
  6           0           0           0           0           0           0           0           0           0           0
  7           0           0           0           0           0           0           0           0           0           0
  8           0           0           0           0           0           0           0           0           0           0
  9           0           0           0           0           0           0           0           0           0           0
 10           0           0           0           0           0           0           0           0           0           0
[0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
  1          38           0           0           0           0           0           0           0           0           0
  2          38           0           0           0           0           0           0           0           0           0
  3          38           0           0           0           0           0           0           0           0           0
  4          38           0           0           0           0           0           0           0           0           0
  5          38           0           0           0           0           0           0           0           0           0
  6          38           0           0           0           0           0           0           0           0           0
  7          38           0           0           0           0           0           0           0           0           0
  8          38           0           0           0           0           0           0           0           0           0
  9          38           0           0           0           0           0           0           0           0           0
 10          38           0           0           0           0           0           0           0           0           0
[0.0] (ok) >               A           B           C           D           E           F           G           H           I           J
 90          38           0           0           0           0           0           0           0           0           0
 91          38           0           0           0           0           0           0           0           0           0
 92          38           0           0           0           0           0           0           0           0           0
 93          38           0           0           0           0           0           0           0           0           0
 94          38           0           0           0           0           0           0           0           0           0
 95          38           0           0           0           0           0           0           0           0           0
 96          38           0           0           0           0           0           0           0           0           0
 97          38           0           0           0           0           0           0           0           0           0
 98          38           0           0           0           0           0           0           0           0           0
 99       10000           0           0           0           0           0           0           0           0           0
[0.0] (ok) > 